app/helpers/mortgage_or_loan_payment_helper.rb

```diff
@@ -0,0 +1,8 @@
+module MortgageOrLoanPaymentHelper
+  def outgoings_links(level_of_help)
```

Comment:
Given that the text and link defined here are identical to the ones defined in the HousingCostsHelper method, you might as well delete this and use the other helper method in both screens.